charts/opentelemetry-demo/values.yaml

```diff
@@ -93,13 +93,15 @@ components:
     # If a different url is desired, set DATABASE_URL here.
     # - name: DATABASE_URL
     #   value:
-    - name: GRPC_PORT
+    - name: FEATURE_FLAG_GRPC_SERVICE_PORT
```

Comment:
Based on the demo's env var file the GRPC port should be 50031 and the http port should be 8081.

https://github.com/open-telemetry/opentelemetry-demo/blob/c5831e8e632259260babdaf7c15423eb2a56d619/.env#L46-L49